icu4c/source/data/BUILDRULES.py

```diff
@@ -457,6 +458,35 @@ def generate_translit(config, io, common_vars):
     ]
 
 
+def generate_brkitr_lstm(config, io, common_vars):
+    input_files = [InFile(filename) for filename in io.glob("brkitr/lstm/*.txt")]
+    dep_files = set(InFile(filename) for filename in io.glob("brkitr/lstm/*.txt"))
+    dep_files -= set(input_files)
+    dep_files = list(sorted(dep_files))
```

Comment:
Suggestion (optional): I think `dep_files` ends up being empty.  You could delete it and pass an empty array for `dep_targets` down below in the RepeatedOrSingleExecutionRequest.

Reply:
DONE